plugins/viaroute.hpp

```diff
@@ -153,15 +154,26 @@ template <class DataFacadeT> class ViaRoutePlugin final : public BasePlugin
         };
         osrm::for_each_pair(phantom_node_pair_list, build_phantom_pairs);
 
-        if (route_parameters.alternate_route && 1 == raw_route.segment_end_coordinates.size())
+        if (false && route_parameters.alternate_route && 1 == raw_route.segment_end_coordinates.size()) //TODO remove false statement

```

Comment:
Logic here should be:
1. No via and no alternative: DirectShortestPath plugin
2. No via and alternative: AlternativePath plugin
3. Via: ShortestPath plugin